File: espritx-Symfony V4/espritx-main/src/Enum/UserStatus.php
<?php

namespace App\Enum;

use App\Traits\RandomizableEnum;
use Elao\Enum\SimpleChoiceEnum;
use ReflectionClass;

/**
 * @extends ReadableEnum<string>
 *
 * @method static ACTIVE
 * @method static PENDING
 * @method static RESTRICTED
 * @method static ALUMNUS
 */
final class UserStatus extends SimpleChoiceEnum
{
  use RandomizableEnum;
  public const ACTIVE = 'active';
  public const PENDING = 'pending';
  public const RESTRICTED = 'restricted';
  public const ALUMNUS = 'alumnus';
}

File: EspritX_Java-Desktop/EspritX_Java-master/src/edu/esprit/enums/UserStatus.java
package edu.esprit.enums;

import edu.esprit.lib.persistence.QueryableEnum;

public enum UserStatus implements QueryableEnum<String> {
    ACTIVE("active"),
    PENDING("pending"),
    RESTRICTED("restricted"),
    ALUMNUS("alumnus");

    private final String value;

    UserStatus(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}
